src/Widgets/FactWidget/FactWidgetEditingConfig.js

```diff
@@ -4,6 +4,20 @@ import factWidgetIcon from "../../assets/images/fact_widget.svg";
 Scrivito.provideEditingConfig("FactWidget", {
   title: "Fact",
   thumbnail: factWidgetIcon,
+  attributes: {
+    key: {
+      options: {
+        allowedTags: ["sub", "sup"],
+        toolbar: ["subscript", "superscript"],
+      },
+    },
+    value: {
+      options: {
+        allowedTags: ["sub", "sup"],
+        toolbar: ["subscript", "superscript"],
+      },
+    },
+  },
```

Comment:
* [ ] Basically same as above. Customers typically ask for sub/sup in headlines. I think `string` is the better default everywhere else. If needed, devs can easily change from string to HTML on their own.